app/assets/stylesheets/normal.css

```diff
@@ -1,19 +1,25 @@
 h1{
-    font-size: 25px;
+    font-size: 30px;
```

Comment:
Unfortunately yes. It was already started with that. I hopefully will do that, when we setup frontend build process.